ldclient/feature_store.py

```diff
@@ -10,24 +10,24 @@ def __init__(self):
         self._initialized = False
         self._features = {}
 
-    def get(self, key):
+    def get(self, key, callback):
```

Comment:
Technically this is an interface change that should require us to change the version to 3.0..